contracts/REXOneWayMarket.sol

```diff
@@ -72,8 +77,7 @@ contract REXOneWayMarket is REXMarket {
     newCtx = ctx;
 
     require(market.oracles[market.outputPools[OUTPUT_INDEX].token].lastUpdatedAt >= block.timestamp - 3600, "!currentValue");
-
-    _swap(market.inputToken, market.outputPools[OUTPUT_INDEX].token, ISuperToken(market.inputToken).balanceOf(address(this)), block.timestamp + 3600);
+    _swap(market.inputToken, market.outputPools[OUTPUT_INDEX].token, ISuperToken(market.inputToken).balanceOf(address(this)), block.timestamp + 360000);
```

Comment:
This isn't good, this is the duration for the swap before it will fail, this saws a swap can linger for 100 hours where before it was saying just an hour